ahmed-yara-sara/src/components/MovieGrid.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import MovieItem from "./MovieItem";
+export default function MovieGrid(props) {
+  const movies = props.movies;
+  console.log(movies);
+
+  return (
+    <>
+      <div className="row mt-3">
+        {movies.map((movie) => (
+          <ul key={movie.id}>
+            <MovieItem
+              image={movie.poster_path}
+              title={movie.title}
+              overview={movie.overview}
+              popularity={movie.popularity}
+              release_date={movie.release_date}
+              
+            />
+          </ul>
```

Comment:
that is really cool that you have movieItem component 
just try to add even the `ul` in it so it will be more clean
what do you think?